assets/js/util/date-range/constants.js

```diff
@@ -26,3 +26,4 @@ export const INVALID_DATE_RANGE_ERROR =
 export const HOUR_IN_SECONDS = 3600;
 export const DAY_IN_SECONDS = 86400;
 export const WEEK_IN_SECONDS = 604800;
+export const HOUR_IN_MILLISECONDS = HOUR_IN_SECONDS * 1000;
```

Comment:
As mentioned, I think we should probably _not_ add this constant at this point, but consider a followup issue to do so across the codebase...

Reply:
Removed, thank you!